crates/stackable-telemetry/src/tracing/settings/mod.rs

```diff
@@ -85,13 +85,20 @@ impl SettingsBuilder {
     }
 
     /// Set specific [`FileLogSettings`].
-    pub fn file_log_settings_builder<P>(self, path: P) -> FileLogSettingsBuilder
+    pub fn file_log_settings_builder<P>(
+        self,
+        path: P,
+        filename_suffix: impl Into<String>,
```

Comment:
note: We mixed two different styles here: `impl` vs generic type parameter. I think we should use a single style here.

Reply:
Ah true! Will fix. Do you have a preference?

Both cases are going to be across multiple lines:

```rust
    pub fn file_log_settings_builder<P, S>(
        self,
        path: P,
        filename_suffix: S,
    ) -> FileLogSettingsBuilder
    where
        P: AsRef<Path>,
        S: Into<String>,
    {
```

and

```rust
    pub fn file_log_settings_builder(
        self,
        path: impl AsRef<Path>,
        filename_suffix: impl Into<String>,
    ) -> FileLogSettingsBuilder {
```